megcruzen/browserify_productOrdering
# Tidied up code, project is complete

# Description
Tidied up code. Project is complete!

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout tidy-up`
3. Run grunt
4. Verify that app still runs correctly.

# Checklist:
